pychromecast/controllers/media.py

```diff
@@ -361,31 +370,51 @@ def skip(self):
         """ Skips rest of the media. Values less then -5 behaved flaky. """
         self.seek(int(self.status.duration)-5)
 
-    def seek(self, position):
+    def seek(self, position, blocking=True, timeout=None):
         """ Seek the media to a specific location. """
-        self._send_command({MESSAGE_TYPE: TYPE_SEEK,
-                            "currentTime": position,
-                            "resumeState": "PLAYBACK_START"})
-
-    def enable_subtitle(self, track_id):
+        self._send_command(
+            {MESSAGE_TYPE: TYPE_SEEK,
+             "currentTime": position,
+             "resumeState": "PLAYBACK_START"},
+            blocking, timeout
+        )
+
+    def enable_subtitle(self, track_id, blocking=True, timeout=None):
         """ Enable specific text track. """
-        self._send_command({
-            MESSAGE_TYPE: TYPE_EDIT_TRACKS_INFO,
-            "activeTrackIds": [track_id]
-        })
+        self._send_command(
+            {MESSAGE_TYPE: TYPE_EDIT_TRACKS_INFO,
+             "activeTrackIds": [track_id]},
+            blocking, timeout
+        )
 
-    def disable_subtitle(self):
+    def disable_subtitle(self, blocking=True, timeout=None):
         """ Disable subtitle. """
-        self._send_command({
-            MESSAGE_TYPE: TYPE_EDIT_TRACKS_INFO,
-            "activeTrackIds": []
-        })
+        self._send_command(
+            {MESSAGE_TYPE: TYPE_EDIT_TRACKS_INFO,
+             "activeTrackIds": []},
+            blocking, timeout
+        )
+
+    def wait(self, timeout=None):
```

Comment:
The name is not representative of what it does. `block_till_status_received` maybe? 

I wonder if we can achieve this by wrapping the status changed bit instead of adding a new threading event?